style.css

```diff
@@ -0,0 +1,98 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif, Arial, Helvetica;
+    box-sizing: border-box;
+}
+
+body{
+    /*background: #001e4d;*/
+    transition: background-color 0.5s;
+    background: linear-gradient(to right, #845EC2, #FF6F91, #FF9671, #F9F871);
+    background-size: 400% 400%;
+    animation: gradient 15s ease infinite;
+}
+@keyframes gradient {
+    0% { background-position: 0% 50%; }
+    50% { background-position: 100% 50%; }
+    100% { background-position: 0% 50%; }
+}
+
+.app{
+    background: #fff;
+    width: 90%;
+    max-width: 600px;
+    margin: 100px auto 0;
+    border-radius: 10px;
+    padding: 30px;
+}
+
+.app h1{
+    font-weight: bold;
+    border-bottom: 1px solid black;
+    font-size: 20px;
+}
+
+h2{
+    font-size: 15px;
+    margin-bottom: 20px;
+}
+
+.quiz{
+    padding: 20px 0;
+}
+.btn{
+    background: #fff;
+    color:#222;   
+    width: 100%;
+    padding: 10px;
+    text-align: left;
+    border-radius: 10px;
+    margin:10px auto 0;
+    
+}
+.btn:hover:not([disabled]){
+    background: #222;
+    color: #fff;
+}
+.btn:disabled{
+    cursor: no-drop;
+}
+
+#btn-next{
+    /* background: #001e4d; */
+    background: #00C9A7;
+    color:#fff;
+    font-weight: 500;
+    border-radius: 10px;
+    border: 0;
+    margin: 20px auto 0;
+    width: 150px;
+    padding:10px;
+    cursor: pointer;
```

Comment:
As one of the first things in my CSS files I would actually be default add `cursor: pointer;` to all `button`'s. It more often than not that you want that style on a button.

Right now the question buttons don't have it and I would recommend they do!